src/python/configure_bindings.py

```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
```

Comment:
as above, my intention is to do it on this PR
I think all scripts now are compatible with Python 3, but let me know if you see anything that should clearly be changed

Reply:
Looks like `code_generator.py` script has python2 string formatting, not sure if python3 supports that? maybe I missed something here

Reply:
@albincorreya I see what you mean. It seems that modulo string formatting still works in Python3, although f-strings are the preferred syntax. If it's okay with you, I think it makes sense to do the switch to f-strings on the `code_generator.py` script on the object-oriented PR that I'm working on, because that will already change significant portions of the script.

Reply:
@jmarcosfer make sense, let's merge this then 💯